docs/rfc/001-backup-format.md

```diff
@@ -0,0 +1,39 @@
+# Structured Backup format
+
+## Principles
+
+- Based-zip64 format (compression level: archive)
+- Based-zip64 standard file encryption
+- Structured tree structure
+
+## General design
+
+```plain
+README.txt
+
+# core file integrity check
+files.json # file hash list with manifest* and README.txt
+
+# the backup metadata
+manifest.json
+  - version = 3?
```

Comment:
We can start with 0 or 1 because it is a binary format.

Reply:
approve, i think you self update the document.